pkg/kubelet/volumes_util.go

```diff
@@ -0,0 +1,79 @@
+package kubelet
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+// AddMountErrorHint performs some basic analysis
+// on the current mount error returned and will
+// add a user hint or resolution tip for enhanced UXP
+func (kl *Kubelet) AddMountErrorHint(volpath string, volname string, inerr error) error{
```

Comment:
Not sure how I feel about this logic being centralized like this.  It feels very much like a cross-cut, reading this logic for different volumes all in the same place.  I don't think it's so bad to keep the logic at the call site it's relevant to.